cmd/gpu_plugin/gpu_plugin_test.go

```diff
@@ -114,16 +126,21 @@ func TestScan(t *testing.T) {
 			expectedMonitors: 1,
 		},
 		{
-			name: "two devices",
+			name: "pf-and-2-vfs",
```

Comment:
Shouldn't it be "2-pfs-and-2-vfs"?

Reply:
No it's one PF (card0) and 2 VFs (card1 & card2).  I'll rename it to "1-pf-and-2-vfs" to be more explicit.

Gah, resolving discarded the comment.